pkg/drivers/vfkit/driver_darwin.go

```diff
@@ -135,6 +143,14 @@ func (d *Driver) Create() error {
 		return fmt.Errorf("%s is an unsupported disk image format", d.ImageFormat)
 	}
 
+	if d.SecondaryDiskSize > 0 {
+		if crcos.FileExists(d.getSecondaryDiskPath()) {
+			return nil
+		}
+		_, err := diskfs.Create(d.getSecondaryDiskPath(), int64(d.SecondaryDiskSize), diskfs.Raw, diskfs.SectorSizeDefault)
```

Comment:
Is `diskfs` required? Could this use `os.Truncate` directly to create an empty disk image?

Reply:
yes, this works, tried using `os.Truncate`, we have to first create an empty file and then change the size using `os.Truncate`